elki/src/main/java/de/lmu/ifi/dbs/elki/distance/distancefunction/strings/LevenshteinDistanceFunction.java

```diff
@@ -104,8 +104,7 @@ public static int levenshteinDistance(String o1, String o2) {
         curr[j + 1] = Math.min(prev[j + 1] + 1, Math.min(curr[j] + 1, prev[j] + cost));
       }
     }
-    final int cost = curr[o2.length()];
```

Comment:
Here, too, I think a named variable `cost` does convey additional information for someone reading the code.